recipes/c-blosc/all/conanfile.py

```diff
@@ -45,21 +45,17 @@ def config_options(self):
 
     def configure(self):
         del self.settings.compiler.cppstd
-        # Snappy is a C++ library with C API. C-Blosc only uses the C API of
-        # Snappy, but if C-Blosc is shared (and snappy static), we have to use
-        # C++ linker.
-        if not (self.options.with_snappy and self.options.shared):
-            del self.settings.compiler.libcxx
+        del self.settings.compiler.libcxx
```

Comment:
```suggestion
        del self.settings.compiler.libcxx
        del self.settings.compiler.cppstd
```

Reply:
`self.settings.compiler.cppstd` is already deleted above

Reply:
Oops indeed!